src/build.py

```diff
@@ -1858,6 +1858,9 @@ def run(sync_filter, build_filter, test_filter):
     Summary()
     return 1
 
+  # Override the default locale to use UTF-8 encoding for all files and stdio
+  # streams (see PEP540), since oure test files are encoded with UTF-8.
+  os.environ['PYTHONUTF8'] = '1'
```

Comment:
Would it make more sense to do this in emscripten itself (i.e. in runner.py)?   Or would that be too late?

Reply:
As far as I can tell there's no supported way to do it once the process has started up. Doing it in runner.py would work for subprocesses but not the parent process. (the test runner does open source files and output e.g. in `run_generated_code` and `do_run_from_file`).  I think there are some situations where there' no supbrocess creation in the test runner.

Reply:
Right.. and are most of the places we are about in the test process, or in the emcc subprocesses?